Documentation/ListModule/UsingEffectively/Index.rst

```diff
@@ -9,56 +9,82 @@ The :guilabel:`Web > List` module allows you to browse through pages and folders
 in your site and view the records that
 are stored there. You can also create and edit records.
 
-Records stored on pages include sub pages, content elements and news articles.
+Select the :guilabel:`Web > List` module and choose a page or folder.
```

Comment:
I woujld call it "storage folder", but in TYPO3 itself this Doktype as really called just "folder". So, it OK.